web/src/OverviewLogPane.tsx

```diff
@@ -212,12 +216,33 @@ export class OverviewLogComponent extends Component<OverviewLogComponentProps> {
     }
   }
 
-  onLogUpdate() {
+  onLogUpdate(e: LogUpdateEvent) {
     if (!this.rootRef.current || !this.cursorRef.current) {
       return
     }
 
-    this.readLogsFromLogStore()
+    let isAffectedByUpdate = false
```

Comment:
hmmmm...this seems like a brittle optimization. is this really needed? AFAIR, the "append" event didn't need this, and the "truncate" event seems infrequent enough that it's not worth the effort.

Reply:
You're right - it's unnecessary. I think at one point I had something else funky going on and thought I needed this. Got rid of it and dramatically simplified the logic here